sci-libs/vegas/vegas-6.0.1.ebuild

```diff
@@ -0,0 +1,22 @@
+EAPI=8
+
+DISTUTILS_EXT=1
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1 pypi
+
+DESCRIPTION="Adaptive multidimensional Monte Carlo integration."
+HOMEPAGE="https://github.com/gplepage/vegas"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+
+RDEPEND="
+	>=dev-python/cython-0.17[${PYTHON_USEDEP}]
+	>=dev-python/numpy-1.16[${PYTHON_USEDEP}]
+	>=sci-libs/gvar-13.0.2[${PYTHON_USEDEP}]
+"
+BDEPEND="${RDEPEND}"
```

Comment:
Ooh right, this will add `cython` as `BDEPEND`. But are `numpy` and `gvar` also `BDEPEND? They should only be BDEPEND if they are executed by the `setup.py`.